ComfyUI/comfy_api_nodes/apis/client.py

```
@@ -773,19 +831,59 @@ def __init__(
+            self.auth_token = auth_kwargs.get(
+                "AUTH_TOKEN_COMFY_ORG", self.auth_token)
+            self.comfy_api_key = auth_kwargs.get(
+                "API_KEY_COMFY_ORG", self.comfy_api_key)
+
+            # Additional fallback: check if auth_token is None but we have the workflow extra_data keys
+            if not self.auth_token:
+                # Try alternative key names that might be used
+                for key in ["AUTH_TOKEN_COMFY_ORG", "auth_token", "comfy_auth_token", "firebase_token"]:
+                    if key in auth_kwargs and auth_kwargs[key]:
+                        self.auth_token = auth_kwargs[key]
+                        print(
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)